tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -920,6 +920,11 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 	})
 
 	ginkgo.Context("[Feature: TLS]", func() {
+		ginkgo.BeforeEach(func() {
```

Comment:
With this change, each TLS case will wait for 1min, I think we can keep the original logic and just wait during the cert-manager creation.

Reply:
reverted.